src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views_component.tsx

```diff
@@ -42,12 +30,12 @@ export const NoDataViewsComponent = ({
   onClickCreate,
   canCreateNewDataView,
   dataViewsDocLink,
+  emptyPromptColor = 'plain',
 }: Props) => {
-  const { euiTheme } = useEuiTheme();
-  const { size } = euiTheme;
-  const maxWidth = `calc(${size.xxl} * 19)`;
+  // Using raw value because it is content dependent
+  const maxWidth = '830px';
```

Comment:
The EuiEmptyPrompt evenly splits the text from the illustration right now with no way to customize it. This caused the title text to wrap lines too soon, so I just manually added a pixel value since it is content based anyway. I'll probably follow up in EUI to see how we can better configure the layout.